src/main/java/smartthings/ratpack/protobuf/renderers/ProtobufRenderer.java

```diff
@@ -24,27 +22,21 @@ public ProtobufRenderer() {
 
     @Override
     public void render(Context ctx, Message message) throws Exception {
-        final String contentType = ctx.getRequest().getHeaders().get("Accept");
-        if (Strings.isNullOrEmpty(contentType) ||
-                JSON.getValue().equals(contentType) ||
-                WILDCARD.getValue().equals(contentType)) {
-            ctx.getResponse().send(JSON.getValue(), printer.print(message));
-        } else if (PROTOBUF.getValue().equals(contentType)) {
-            ctx.getResponse().send(contentType, message.toByteArray());
-        } else {
-            render406(ctx, contentType);
+        String accept = ctx.getRequest().getHeaders().get("Accept");
+        ratpack.http.MediaType mediaType = DefaultMediaType.get(accept);
+        if (mediaType.isJson()) {
+            json(ctx, message);
+            return;
         }
+        protobuf(ctx, message);
+    }
```

Comment:
Could've also went w/ Ratpack's `ctx.byContent` method for media type negotiation.  The reason I didn't was because it does not support "+json" / "+protobuf" syntax.